components/01-atoms/images/image/figure.twig

```diff
@@ -0,0 +1,32 @@
+{% set figure__base_class = 'figure' %}
+{% set image__link__base_class = image__link__base_class|default('link') %}
+
+{% set figure__attributes = {
+  class: bem(figure__base_class, figure__modifiers, figure__blockname),
+} %}
+
+{% if figure__caption %}
+  {% set figure__attributes = figure__attributes|merge({
+    'aria-label': figure__caption,
+  }) %}
+{% endif %}
+
+<figure {{ add_attributes(figure__attributes) }}>
+  {% if image__url %}
```

Comment:
@kara-franco here we have the ability to link the `<img>` within the figure component. Should this `<a>` be required to have a title attribute or is the required alt attribute on the `<img>` make the link self-descriptive enough?

Reply:
@ModulesUnraveled I'll approve once we get confirmation from Kara regarding my accessibility question above.

Reply:
Thanks for the info! TIL